videomaker/inc/patterns/pattern-grid.php

```diff
@@ -18,8 +18,8 @@
 	</div></figure>
 	<!-- /wp:embed -->
 
-	<!-- wp:heading {"level":4} -->
-	<h4>' . esc_html__( 'Man Working at Dawn', 'videomaker' ) . '</h4>
+	<!-- wp:heading {"level":4,"style":{"spacing":{"margin":{"bottom":"calc(0.5 * var(--wp--style--block-gap))"}}}} -->
+	<h4 style="margin-bottom:calc(0.5 * var(--wp--style--block-gap))">' . esc_html__( 'Man Working at Dawn', 'videomaker' ) . '</h4>
```

Comment:
Should we be calculating values for things in block markup?

I know it works, and we've done it elsewhere, and it helps makes things consistent.  However, the value can't be expressed to the user in that form (it's just blank in the Block details panel). 

I think it makes sense to use calc() and var() in CSS and in theme.json definitions, but I'm not convinced it's the most bestest in block markup.

Thoughts?

Reply:
Ah yes, I agree, and I didn't realize how this was expressed to the user. I guess even if these values didn't show as blank in the details panel, they're still a lot more confusing to read than a fixed value.

I'll update this PR and remember this for the future. Thanks!

Reply:
It's not just you @mikachan!  I've been using them too; that's what they were made for after all. :P

I'm glad we've got consensus on limiting their use though.